webview/src/shared/components/contextMenu/ContextMenu.tsx

```diff
@@ -58,6 +60,8 @@ export const ContextMenu: React.FC<ContextMenuProps> = ({
     placement={'bottom'}
     interactive
     onTrigger={positionContextMenuAndDisableEvents}
+    onClickOutside={(instance: Instance) => instance.hide()}
```

Comment:
[F] If you click outside of the row hide the instance and make a new one

Reply:
A bug in Tippy stops this from working with `@testing-library/react`. I have spent ~1.5 hrs and cannot find a way around it. Two clicks in the storybook show the bug as well. IRL it works 😕.